jekyll/_cci2/ssh-access-jobs.md

```diff
@@ -84,6 +84,8 @@ If you have multiple accounts, double-check that you are authenticated as the ri
 
 If you are authenticating as the wrong user, you can probably resolve this by offering a different SSH key with `ssh -i`. See the next section if you need a hand figuring out which key is being offered.
 
+The user that triggered the rerun is the user that is required to authenticate. You will need to make sure that your username is in the 'Enable SSH' step.
```

Comment:
Would it make sense to add this paragraph at the beginning of the section instead, i.e. right after the "Ensure authenticating..." heading? That would make the first part of the section be more cohesive re: authenticating correctly, and then the second half is about what happens if you authenticated incorrectly. Let me know if I'm misunderstanding this bit.

Reply:
@kbatuigas completely agree! I just wasn't sure where to fully put it. I've updated my commit to add it at the top of that paragraph. 🙇